tests/bsim/bluetooth/host/misc/sample_test/src/data.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_TESTS_BSIM_BLUETOOTH_HOST_MISC_SAMPLE_TEST_SRC_DATA_H_
+#define ZEPHYR_TESTS_BSIM_BLUETOOTH_HOST_MISC_SAMPLE_TEST_SRC_DATA_H_
+
+#include <zephyr/bluetooth/uuid.h>
+
+static uint8_t payload_1[] = {0xab, 0xcd};
+static uint8_t payload_2[] = {0x13, 0x37};
+
+BUILD_ASSERT(sizeof(payload_1) == sizeof(payload_2));
```

Comment:
Given the nature of this test, suggest to add a description for the BUILD_ASSERT